configure

```diff
@@ -44,8 +44,8 @@ while getopts ":ho:-:p" opt; do
 	    # process --long-options
 	    case "$OPTARG" in
                 help)  usage ;;
-                *=*)   opt=${OPTARG%%=*}; val=${OPTARG#*=}
-                       eval "$opt=$val" ;;
+                *=*)   opt=${OPTARG%=*}; val=${OPTARG#*=}
```

Comment:
Yeah, kind of a typo because I initially tried to do the `s/-/_/` substitution here and forgot to restore it properly.